src/lib/uTop_main.ml

```diff
@@ -79,6 +79,39 @@ let index_of_offset src ofs =
 
 let convert_locs str locs = List.map (fun (a, b) -> (index_of_offset str a, index_of_offset str b)) locs
 
+let convert_loc str (a, b) = (index_of_offset str a, index_of_offset str b)
```

Comment:
If this function is introduced then `convert_locs` should use it.

Reply:
Ack